### PR TITLE
Communicating to the user if  the server is not available

### PR DESCRIPTION
**Pull request by Kledis98**
This was done by :
1- Disabling the editor.
2-Changing the placeholder text.

### CODE REVIEW

File: src/app/home/home.component.ts [hunk 1]
```diff
@@ -68,9 +68,17 @@ export class HomeComponent implements AfterViewInit, OnDestroy {
...
         this.initializeURLs();
         // should any other call be made here? probably not... actually even this should be removed soon
+        let response = this.http.get(this.pingURL).subscribe(function(){
+            if(!response.ok){
+                document.getElementById('editor').disabled = true;
+                document.getElementById('editor-placeholder').innerText = 'The editor is out of service for now '
+                
+            }
+        
+        })
```
Comment (AndiBraimllari) [inline src/app/home/home.component.ts:78]: Could you instead try 
```
this.http.get(this.pingURL).subscribe(() => {}, () => {
    // TODO add logic here
});
```

Basically this subscribe method (currently referring to a deprecated version which should be enough for now) can take three functions as parameters, with the first being called on its next value, the second is called on an error (of interest to us), and the third on completion. This link might provide some more insight on this, https://rxjs.dev/deprecations/subscribe-arguments

File: src/app/home/home.component.ts [hunk 2]
```diff
@@ -68,9 +68,17 @@ export class HomeComponent implements AfterViewInit, OnDestroy {
...
         this.http.get(this.pingURL).subscribe(() => {
-            console.log('pinging server...');
-        });
+            console.log('pinging server...')
+            });
```
Comment (AndiBraimllari) [inline src/app/home/home.component.ts:81]: Please undo these changes as they're currently unrelated to the scope of your changes.

File: src/app/home/home.component.ts (hunk 1 above)
Comment (AndiBraimllari) [reply src/app/home/home.component.ts:78]: I.e. the first method wouldn't do anything, and in the second method you can try adding the disabling of the editor and the text change of the placeholder.
We can refactor this later.

**Comment by AndiBraimllari** [comment]
After applying the suggested changes I believe the pipeline should go green.
Feel free to push more code and we can discuss again.

**Comment by AndersonCeci** [comment]
I also suggest you create a branch instead of working on the main branch

**Comment by AndersonCeci** [comment]
> Can you @AndersonCeci create an issue for adding tests to such functionality? We can then address it at another point in time.

Done #172

**Comment by AndiBraimllari** [comment]
Nice, the pipeline is green!
Can you run `npm run prettier:fix` to format the code?
Also, there's a change in angular.json, can you revert that? Unsure if you've made that change manually or if it was done automatically.

**Comment by AndiBraimllari** [comment]
LGTM, merging now!
